platform/test/spec/testActivityManagerSpec.js

```diff
@@ -6,91 +6,67 @@ import { ActivityConfigValidator } from "../../src/ActivityConfigValidator.js";
+        })
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+        it("initialises the configValidator property", () => {
+            expect(am.configValidator).toBeInstanceOf(ActivityConfigValidator);
+        })        
 
-        // Check the expected results
-        expect(am.fileHandler).toBe(fileh);
-    })  
+        it("initialises the accessPanelDef property using param panelDefAccessor", () => {
+            expect(am.accessPanelDef).toBe(refPanelDef);
+        })  
 
-    it("constructor -  uses the query string to set the activitiesUrl property", () => {
-        // Setup
-        const ACTIVITY_URL="test://a.url";
-        const QUERY = "?activities=" + ACTIVITY_URL;
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        it("initialises the fileHandler property  using param fileHandler", () => {
+            expect(am.fileHandler).toBe(fileh);
+        })  
 
-        spyOn( utility, "getWindowLocationSearch").and.returnValue(QUERY);
+        it("uses the query string to set the activitiesUrl property", () => {
+            const QUERY = "?activities=" + ACTIVITY_URL;
+            spySearch.and.returnValue(QUERY); //Update the spy
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+            am = new ActivityManager(refPanelDef, fileh);
 
-        // Check the expected results
-        expect(utility.getWindowLocationSearch).toHaveBeenCalled();
-        expect(am.activitiesUrl).toBe(ACTIVITY_URL);
-    })
+            expect(utility.getWindowLocationSearch).toHaveBeenCalled();
+            expect(am.activitiesUrl).toBe(ACTIVITY_URL);
+        })
 
-    it ("constructor -  if the current activity is provided in the url query string, set the activityId property", () => {
-        // Setup
-        const ACTIVITY_URL="test://a.url";
-        const ACTIVITY_ID="a1"
-        const QUERY = "?"+ ACTIVITY_ID + "=&" + "activities=" + ACTIVITY_URL;
-        const fileh = new FileHandler("test://th.url") 
-        const refPanelDef = () => {};
+        it ("if the current activity is provided in the url query string, set the activityId property", () => {
+            const ACTIVITY_ID="a1"
+            const QUERY = "?"+ ACTIVITY_ID + "=&" + "activities=" + ACTIVITY_URL;
+            spySearch.and.returnValue(QUERY); // Update the spy
 
-        spyOn( utility, "getWindowLocationSearch").and.returnValue(QUERY);
+            am = new ActivityManager(refPanelDef , fileh);
 
-        // Call the target object
-        var am = new ActivityManager(refPanelDef , fileh);
+            expect(am.activityId).toBe(ACTIVITY_ID);
+        })
 
-        // Check the expected results
-        expect(am.activityId).toBe(ACTIVITY_ID);
     })
 
     it("initializeActivities - causes the activity file to be fetched from its URL using the fileHandler", () => {
```

Comment:
Shouldn't these be in the next `describe` as they are about activity initialisation?

Reply:
Yes the test style evolved slightly as they were added it makes sense to include them in the same describe block. Fixed in 023439982f88091a37a89567f047fcb5495458bd